src/screens/Services/Services.js

```diff
@@ -282,7 +272,7 @@ class ServicesScreen extends React.Component<Props> {
           const address = this.getCryptoPurchaseAddress();
           if (address === null) return;
           this.props.navigation.navigate(SENDWYRE_INPUT, {
-            onSubmit: async (values: SendwyreTrxValues) => {
+            onSubmit: async (values: {}) => {
```

Comment:
Is this typing removal actually useful?

Reply:
Changes

Reply:
Do we need to pass onSubmit callback as navigation param? Is it going to change depending on navigation call? If not then please move the action to here.


Yes, because we are reusing method: tryOpenCryptoPurchaseUrl present in Services screen